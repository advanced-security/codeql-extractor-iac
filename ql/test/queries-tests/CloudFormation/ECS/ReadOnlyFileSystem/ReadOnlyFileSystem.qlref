security/CloudFormation/ECS/ReadOnlyRootFileSystem.ql