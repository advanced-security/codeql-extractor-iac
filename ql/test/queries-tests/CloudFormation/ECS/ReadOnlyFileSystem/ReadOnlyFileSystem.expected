| ecs.yml:11:7:40:2 | CloudFormation Resource Properties | Containers must have explictly only read only root filesystem |